src/runge-kutta.jl

```diff
@@ -0,0 +1,315 @@
+# This file contains the implementation of explicit Runkge-Kutta
+# solver from (Hairer & Wanner 1992 p.134, p.165-169).
+
+include("tableaus.jl")
+
+# intermediate level interface
+
+"""
+
+A general Runge-Kutta stepper (it cen represent either, a fixed step
+or an adaptive step algorithm).
+
+"""
+immutable RKStepper{Kind,tab_name,T<:Number} <: AbstractStepper
+    tableau::Tableau
+    function RKStepper()
+        tab = eval(tab_name)
```

Comment:
Combining the suggestions is good: leave the tableaus alone (at least for now) but collect them in a Dict. 

I think the reason I made the `Name` a type parameter was to allow someone to write a specific stepper for a specific RK method, which has not happened yet and may never will.  Nonetheless, I think it's ok and Julian to provide that possibility.  I'm not sure why `S` became a parameter, probably should be removed at some point.